src/linear_algebra/alternating.lean

```diff
@@ -301,13 +325,13 @@ namespace alternating_map
 
 /-- Alternatizing a multilinear map that is already alternating results in a scale factor of `n!`,
 where `n` is the number of inputs. -/
-lemma to_multilinear_map_alternization [fintype ι] (a : alternating_map R M N' ι) :
-  a.to_multilinear_map.alternatization = nat.factorial (fintype.card ι) • a :=
+lemma coe_alternization [fintype ι] (a : alternating_map R M N' ι) :
```

Comment:
Should this be `coe_alternatization`? (Are there other `alternization` / `alternatization` inconsistencies?)

Reply:
I think this is the only inconsistency so far, nice catch.
```suggestion
lemma coe_alternatization [fintype ι] (a : alternating_map R M N' ι) :
```